src/options.c

```diff
@@ -272,6 +273,8 @@ static struct opt_entry opt_list[] = {
 "--single-user-seed=FILE    wordlist with seeds per username (user:password[s]\n" \
 "                           format)\n" \
 "--[no-]single-retest-guess override config for SingleRetestGuess\n" \
+"--single-pair-max=N        override max. number of word pairs generated (%u)\n" \
+"--no-single-pair           disable single word pair generation\n" \
```

Comment:
Maybe it'd be cleaner and would save us a line in the `--help` output if we simply accepted `--single-pair-max=0` as meaning the same as `--no-single-pair` (perhaps we already do with this commit), without having the latter option name. I am impressed by what the new tri-state code is capable of, but that doesn't necessarily make this a good idea here.

Reply:
Yeah maybe. It'd be a usage-blob-change only - the code already allows this